tests/cc/test_clang.py

```diff
@@ -172,5 +172,40 @@ def test_char_array_probe(self):
     return 0;
 }""")
 
+    def test_probe_read_helper(self):
+        b = BPF(text="""
+#include <linux/fs.h>
+static void print_file_name(struct file *file) {
```

Comment:
would  be nice to add few more tests for this to cover the rest of possibilities that rewriter already supports.